src/test/java/hdfs/jsr203/TestFileSystem.java

```diff
@@ -310,4 +311,29 @@ public void getRootDirectories() throws IOException {
 
         assertNotNull(pathToTest.getFileSystem().getRootDirectories());
     }
+    
+    @Test
+    public void testCopyeAndMoveFiles() throws IOException {
+        URI uriSrc = clusterUri.resolve("/tmp/testSrcFile");
+        Path pathSrc = Paths.get(uriSrc);
+        
+        URI uriDstCp = clusterUri.resolve("/tmp/testDstCopyFile");
+        Path pathDstCp = Paths.get(uriDstCp);
+        
+        BufferedWriter br = Files.newBufferedWriter(pathSrc);
```

Comment:
Maybe I use jdk 1.8 Jdk1.7 environment. On my computer the BufferedWriter works. 
I change it to OutputStream and it's ok.